experiments/apriltag/src/apriltag_truth/scripts/truth.py

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/env python2
+# Estimate ground truth poses from apriltag detections
+# Ethan Yang ethany@mit.edu
+
+import time
+
+import numpy as np
+import rospy
+from apriltag_ros.msg import AprilTagDetectionArray
+from scipy.spatial.transform import Rotation as Rot
+
+
+class TruthEvaluate(object):
+    def __init__(self):
+        # all measurements in m
+        self.tag_size = 0.05
+        self.margin = 0.01
+        self.paper_height = 279.4 * 10**-3
+        self.paper_width = 215.9 * 10**-3
+
+        bag_name = rospy.get_param('bag_name', None) + '_gt.txt'
+        obj_type = bag_name[:bag_name.index('_')]
+        assert obj_type in ['cracker', 'spam', 'sugar']
+        if obj_type == 'cracker':
+            # Tags are in order [bottom left, bottom right, middle left, etc.]
+            self.tags = np.array([8, 9, 10, 11, 12, 13])
```

Comment:
Maybe param this (and other parameters) out in the future? Like adding them as launch file args.